src/entries/Entry.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import firebase from 'firebase';
+import { format } from 'date-fns';
+import { View, TextInput } from 'react-native';
+import { AddInputButton } from './AddInputButton';
+import { ItemList } from './EntryList';
+
+
+export class Item extends Component {
```

Comment:
lets rename Item to something more meaningful. I remember you had some good proposal yesterday